docs/maintainers/release.md

```diff
@@ -1,44 +1,78 @@
+     all significant changes and all bug fixes since the first version of the
+     previous minor release should be mentioned, even bug fixes which have
+     already been included in some patch release. *For a patch release*, you
+     will mention all the bug fixes since the previous release with the same
+     minor version. The commit message must be *exactly* `"Update CHANGELOG for
+     <TAG> release"`, as a bot will look for this commit and cherry-pick it to
+     update the main branch (starting with Antrea v1.0). The
+     [process-changelog.go](../../hack/release/process-changelog.go) script may
+     be used to easily generate links to PRs and the Github profiles of PR
+     authors.
+   - a commit to update [VERSION](../../VERSION) as needed, using the following
+     commit message: `"Set VERSION to <TAG>"`.
+
+3. Run all the tests for the PR, investigating test failures and re-triggering
+   the tests as needed.
+   - Github worfklows are run automatically whenever the head branch is updated.
+   - Jenkins tests need to be [triggered manually](../../CONTRIBUTING.md#getting-your-pr-verified-by-ci).
+   - Cloud tests need to be triggered manually through the
+     [Jenkins web UI](https://jenkins.antrea-ci.rocks/). Admin access is
+     required. For each job (AKS, EKS, GKE), click on `Build with Parameters`,
+     and enter the name of your fork as `ANTREA_REPO` and the name of your
+     branch as `ANTREA_GIT_REVISION`. Test starting times need to be staggered:
+     if multiple jobs run at the same time, the Jenkins worker may run
+     out-of-memory.
+
+4. Request a review from the other maintainers, and anyone else who may need to
+   review the release notes. In case of feedback, you may want to consider
+   waiting for all the tests to succeed before updating your PR. Once all the
+   tests have run successfully once, address review comments, get approval for
+   your PR, and merge.
+   - this is the only case for which the "Rebase and merge" option should be
+     used instead of the "Squash and merge" option. This is important, in order
+     to ensure that changes to the CHANGELOG are preserved as an individual
+     commit. You will need to enable the "Allow rebase merging" setting in the
+     repository settings temporarily, and remember to disable it again right
+     after you merge.
+
+5. Make the release on Github **with the release branch as the target** and copy
+   the relevant section of the CHANGELOG as the release description (make sure
+   all the markdown links work). You typically should **not** be checking the
+   `pre-release` box. This would only be necessary for a release candidate
+   (e.g., `<TAG>` is `1.4.0-rc.1`), which we do not have at the moment. There is
+   no need to upload any assets as this will be done automatically by a Github
+   workflow, after you create the release.
+
+6. After a while (time for the Github workflows to complete), check that:
+   - the docker image has been pushed to
```

Comment:
I found the workflow in Antrea repo didn't upload the antrea-ubuntu image after all workflows completed. And it's uploaded after 1 hour. Guess there is another workflow in other place doing it. Should it be mentioned so other release maintainers wouldn't think there were something wrong.

Reply:
Yes my bad. I'll update the documentation. You can check out these workflows: https://github.com/vmware-tanzu/antrea-build-infra/actions/workflows/march-build.yml

Reply:
Thanks for noticing this, I added a new doc to explain what's going on